tools/cibadmin.c

```diff
@@ -119,6 +131,9 @@ static struct crm_option long_options[] = {
     {"-spacer-",    0, 0, '-', " $EDITOR $HOME/local.xml", pcmk_option_example},
     {"-spacer-",    0, 0, '-', " cibadmin --replace --xml-file $HOME/local.xml", pcmk_option_example},
 
+    {"-spacer-",    0, 0, '-', "Render configuration in color per respective ACLs as evaluated for user tony:", pcmk_option_paragraph},
+    {"-spacer-",    0, 0, '-', " cibadmin -L -U tony", pcmk_option_example},
+
```

Comment:
This needs to be guarded by ENABLE_ACL as well

Reply:
Oh yeah